src/types/api/cluster.ts

```diff
@@ -38,3 +38,18 @@ export interface TClusterInfo {
     Balancer?: string; // additional
     Solomon?: string; // additional
 }
+
+export interface TClusterConfigFeatureFlag {
+    Name: string;
+    Current?: boolean;
+    Default?: boolean;
+}
+
+export interface TClusterConfigDb {
+    Name: string;
+    FeatureFlags: TClusterConfigFeatureFlag[];
+}
+
+export interface TClusterConfigs {
+    Databases: TClusterConfigDb[];
+}
```

Comment:
In `types/api` types are grouped by endpoints or entites. Please, create a separate file for configs, they don't connected with `TClusterInfo` - `/viewer/cluster` response